### PR TITLE
add icon parameter to the alert rpc

### PR DESCRIPTION
**Pull request by iCollin**
implements #101

### CODE REVIEW

File: MOBILE_API.xml [hunk 1]
```diff
@@ -4902,6 +4902,13 @@
                 If omitted on supported displays, the displayed alert shall not have any SoftButtons.
             </description>
         </param>
+
+        <param name="alertIcon" type="Image" mandatory="false" since="6.0">
```
Comment (jacobkeeler) [inline MOBILE_API.xml:4910]: Needs to be added to `ImageFieldName` as well

**Review by jacobkeeler — CHANGES_REQUESTED** [review]

File: MOBILE_API.xml (hunk 1 above)
Comment (iCollin) [reply MOBILE_API.xml:4910]: added in c126332

Comment (jacobkeeler) [reply MOBILE_API.xml:4910]: Need to run RPCParser again to update README

Comment (iCollin) [reply MOBILE_API.xml:4910]: ran again in [459b3c7](https://github.com/smartdevicelink/rpc_spec/pull/180/commits/459b3c7b5b2aad4ff56d5d896b2931ca42165726)

**Review by jacobkeeler — APPROVED** [review]

**Pull request merged**